CMake/SEMMacroBuildCLI.cmake

```diff
@@ -203,6 +203,20 @@ macro(SEMMacroBuildCLI)
     ARCHIVE_OUTPUT_DIRECTORY "${LOCAL_SEM_ARCHIVE_OUTPUT_DIRECTORY}"
     )
 
+  # Enable Interprocedural / global link-time optimization, if supported
+  if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.13.0)
```

Comment:
Instead of using `VERSION_GREATER_EQUAL` (it is not available in CMake 3.5), consider using `VERSION_GREATER` and `VERSION_EQUAL`. 

An alternative would be to use something like this:

```
include(CheckIPOSupported OPTIONAL RESULT_VARIABLE module_found)
if(module_found)
  ...
endif()
```

Reply:
> Instead of using VERSION_GREATER_EQUAL (it is not available in CMake 3.5 set as minimum required version),

Done.

> include(CheckIPOSupported OPTIONAL RESULT_VARIABLE module_found)

This is not the same -- the `CheckIPOSupported` has been around for a while, but it was not supported for Visual Studio until CMake 3.13.0.

Reply:
Gotcha. Thanks for updating then.